generators/server/prompts.js

```diff
@@ -157,7 +157,7 @@ function askForServerSideOpts(meta) {
             }
         },
         {
-            when: response => applicationType === 'microservice' || (response.authenticationType === 'uaa' && applicationType === 'gateway'),
+            when: response => applicationType === 'microservice' || ((response.authenticationType === 'uaa' || response.authenticationType === 'oauth2') && applicationType === 'gateway'),
```

Comment:
Cassandra is not supported for oauth so may be you should just add the `No database` option to the next block specifc for oauth2

Reply:
OK, I'll do it. thanks

Reply:
Why is Cassandra not supported with OAuth? I believe the previous implementation required database tables, where as the new implementation does not.

Reply:
No idea for Cassandra!

I made the changes. Can I please submit the changes made by doing this on my branch:
git commit -a
git rebase master -i
git push -f

Reply:
@deepu105 It's done.
@mraible No idea for Cassandra !

Reply:
@mraible I dont know I just inferred from current code and thought it was from you. But if you say it works then it works :)